index.js

```diff
@@ -55,6 +56,7 @@ var anymatch = function(criteria, value, returnIndex, startIndex, endIndex) {
     if (path.sep === '\\' && typeof string === 'string') {
       altString = string.split('\\').join('/');
```

Comment:
split-join is sometimes faster. it depends on the length of the string etc. 

> And why would a windows user ever supply a path with a leading backslash?

they might not intentionally, but node's `path.normalize()` will turn forward slashes into back slashes.

> but I could use normalize-path directly to avoid that problem, right?

yes, that should work.